### PR TITLE
Fixes broken git bash prompt by using LF

### PR DESCRIPTION
**Pull request by jotbe**

### CODE REVIEW

**Comment by jotbe** [comment]
Please do not merge yet as I encountered a weird bug resulting in the output `: command not found` when starting a Bash session. This bug occured after I ran `update-vm.sh` from within the bugfix branch.

File: cookbooks/vm/files/default/git_ps1 [hunk 1]
```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
+
```
Comment (tknerr) [inline cookbooks/vm/files/default/git_ps1]: was it just another LF / CRLF issue in the end?

**Comment by tknerr** [comment]
Is this ready to be merged?

**Comment by tknerr** [comment]
@jotbe I couldn't reproduce this on my end on macOS.

So I assume the issue is that if you are cloning the linus-kitchen repo on Windows, Git would convert the LF in https://github.com/tknerr/linus-kitchen/blob/master/cookbooks/vm/files/default/git_ps1 to CRLF on checkout already.

Can you try if adding a .gitattributes file like in https://github.com/tknerr/linus-kitchen/blob/master/scripts/.gitattributes resolves the issue?

**Comment by tknerr** [comment]
I created PR #86 to add a .gitattributes file, which should fix the issue. Can't test it though due to lack of a windows installation ;-)

**Pull request closed without merging**